src/main/kotlin/com/zepben/evolve/services/network/tracing/feeder/AssociatedTerminalTrace.kt

```diff
@@ -22,13 +22,6 @@ import com.zepben.evolve.services.network.tracing.traversals.BasicTraversal
  */
 internal object AssociatedTerminalTrace {
 
-    /**
-     * @return a traversal ignoring open points.
-     */
-    internal fun newTrace(): BasicTraversal<Terminal> {
-        return newTrace(OpenTest.IGNORE_OPEN)
-    }
```

Comment:
Never used, since `AssignTo(Lv)Feeders` runs separate traces for normal and current network state. Can be added back in the future if required.